File: front/webav360/src/app/Models/Dashboard/AlunoDashboard.ts
import { CriterioAlunoDashboard } from './CriterioAlunoDashboard';

export interface AlunoDashboard {
  alunoId: number;
  nome: string;
  media: number;
  grupoId: number,

  criterioAluno: CriterioAlunoDashboard[];

  avaliou: boolean;
  totalNotas: number;
}

File: back/Application/DTOs/AlunoDashboard.cs
namespace Application.DTOs;

public class AlunoDashboardDTO
{
    public int AlunoId { get; set; }
    public string Nome { get; set; } = null!;
    public decimal Media { get; set; }
    public int GrupoId { get; set; }
    public List<CriterioAlunoDashboardDTO> CriterioAluno { get; set; } = [];
    public bool Avaliou { get; set; }
    public int TotalNotas { get; set; }
}